client/src/components/Utils/Button/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+export default function Button({ className }) {
+  return (
+    <button type="button" className={`button ${className}`}>
+      Enter
```

Comment:
The content of the button varies from one place to another.

So in order to make your component's content dynamic, you need to receive the content in the props.